### PR TITLE
Run tests on Github Actions

### PR DESCRIPTION
**Pull request by dnkolegov**
This PR improves Github Actions flow:

- It adds prettier and solhint linters
- Adapts the code to prettier format
- Triages slither findings
- Adapats the makefile for old-school engineers

The checks will be green if the previous PRs are merged.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,18 +1,34 @@
...
 check-subnet:
 	docker run --rm -v $(shell pwd):/app -w /app mythril/myth:latest -v4 analyze --solc-json remappings.json ./src/SubnetActor.sol --solv 0.8.19
 
-deploy-ipc:
-	./ops/deploy.sh $(NETWORK)
+# ==============================================================================
+# Development support
+
+lint:
+	solhint 'src/**/*.sol'
```
Comment (adlrocha) [inline Makefile:25]: I think the Limechain team was using `forge lint` as their base linter. I don't have a strong opinion here, so I am fine using `solhint`

**Review by adlrocha — APPROVED** [review]
LGTM. I haven't reviewed in depth the changes in the `.sol` files as they all seem `fmt`-related, but let me know if I should check them.

File: Makefile (hunk 1 above)
Comment (dnkolegov) [reply Makefile:25]: I could not find `forge lint`. I found only that - https://github.com/foundry-rs/foundry/issues/1970

Comment (adlrocha) [reply Makefile:25]: That's fine, let's use solhint which is actually the one that IDEs use as default.

Comment (dnkolegov) [reply Makefile:25]: We can add more linters if needed or replace that one if necessary later.

**Pull request merged**